spec/rack/test_spec.rb

```diff
@@ -165,7 +165,7 @@ def close
 
       it "closes response's body" do
         body = CloseableBody.new
-        expect(body).to receive(:close)
+        expect(body).to receive(:close).at_least(:once)
```

Comment:
Since `rack v2.2.0`,`close` is called automatically - which was making this test fail since we might now be calling it twice:

https://github.com/rack/rack/commit/b058947745d3a1bf32e5f08b96409c2aaabc2636